plugins/techdocs-backend/src/service/router.ts

```diff
@@ -140,7 +135,7 @@ export async function createRouter({
     res.redirect(`${storageUrl}${req.path.replace('/docs', '')}`);
   });
 
-  if (publisher instanceof LocalPublish) {
+  if (publisher.isLocalPublisher()) {
```

Comment:
Shouldn't just the PublisherBase have a `docsRouter` method that has the same return type as `express.static`? I mean all types of target will need to support accepting file requests. And then you won't have to check the `isLocal` stuff; the router won't have to know HOW the requests are being fulfilled

Reply:
Yep, that will be excellent! Abstracted away this logic from the router to the Publisher.